controllers/platformnetwork_controller.go

```diff
@@ -1325,3 +1333,51 @@ func (r *PlatformNetworkReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&starlingxv1.PlatformNetwork{}).
 		Complete(r)
 }
+
+// Verify whether we have annotation restore-in-progress
+func (r *PlatformNetworkReconciler) checkRestoreInProgress(instance *starlingxv1.PlatformNetwork) bool {
+	restoreInProgress, ok := instance.Annotations[cloudManager.RestoreInProgress]
+	if ok && restoreInProgress != "" {
+		return true
+	}
+	return false
+}
+
+// Update status and clear annoatation
+func (r *PlatformNetworkReconciler) RestorePlatformNetworkStatus(instance *starlingxv1.PlatformNetwork) error {
+	r.ReconcilerEventLogger.NormalEvent(instance, common.ResourceUpdated, "Status without reconcilation: %t", instance.Status.InSync)
```

Comment:
Can we rephrase this:
`"Status updated for PlatformNetwork resource '%s' during restore with following values: Reconciled=%t InSync=%t DeploymentScope=%s", instance.Status.Reconciled, instance.Status.InSync, instance.Status.DeploymentScope`

and we can place this `r.ReconcilerEventLogger.NormalEvent` after `err = r.Client.Status().Update` when err == nil

Reply:
Changes are done